game/hud/src/services/session/layout.ts

```diff
@@ -86,20 +92,26 @@ export function saySomething(s: string) : LayoutAction {
   }
 }
 
+export function adjustWidgetPositions() {
+  return {
+    type: ADJUST_POSITIONS,
+  }
+}
 
 
-let defaultWidgets = () => {
+const defaultWidgets = (): any => {
```

Comment:
Use function dec's instead of expresisons

https://github.com/airbnb/javascript#functions--declarations
https://github.com/airbnb/javascript#naming--self-this